backend/polls/views.py

```diff
@@ -118,6 +128,104 @@ def stats(self, *args, **kwargs):
             }
         )
 
+    @action(detail=False, methods=("get",), permission_classes=(PollStatsPermission,))
+    def statistics(self, *args, **kwargs):
+
+        victories_by_user = {}
+        defeats_by_user = {}
+        participations_by_user = {}
+
+        set_cache = False
+
+        if (
+            cache.get("participations") is not None
+            and cache.get("victories_ranking") is not None
+            and cache.get("defeats_ranking") is not None
+        ):
+            # Normally there should all be None or all be defined but just in case...
+            participations_by_user = cache.get("participations")
+            victories_by_user = cache.get("victories_ranking")
+            defeats_by_user = cache.get("defeats_ranking")
+
+        else:
+            # Computing the rankings...
+
+            for poll in self.queryset:
+                if poll.has_been_published:
+                    pollDate = poll.publication_date
```

Comment:
```suggestion
                    poll_date = poll.publication_date
```
We try to use `snake_case` for python and `camelCase` for TS